plugin/trino-kafka/pom.xml

```diff
@@ -98,11 +98,21 @@
             <artifactId>protobuf-java-util</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>io.confluent</groupId>
+            <artifactId>kafka-avro-serializer</artifactId>
+        </dependency>
+
         <dependency>
             <groupId>io.confluent</groupId>
             <artifactId>kafka-schema-registry-client</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>io.confluent</groupId>
+            <artifactId>kafka-schema-serializer</artifactId>
```

Comment:
cannot be used in compile scope due to license issues

Reply:
nevermind, those two modules are Apache licensed. It's so confusing with Confluent stuff. 😭